plugins/browser-plugin-ga-cookies/src/index.ts

```diff
@@ -28,27 +28,67 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import { SelfDescribingJson } from '@snowplow/tracker-core';
+import { LOG, SelfDescribingJson } from '@snowplow/tracker-core';
 import { BrowserPlugin, cookie } from '@snowplow/browser-tracker-core';
-import { Cookies } from './contexts';
+import { GA4Cookies, UACookies } from './types';
+import { GOOGLE_ANALYTICS_4_COOKIES_SCHEMA, UNIVERSAL_ANALYTICS_COOKIES_SCHEMA } from './schemata';
+
+interface GACookiesPluginOptions {
+  ua?: boolean;
+  ga4: boolean;
+  ga4ContainerId: string;
+}
+
+const defaultPluginOptions = {
+  ua: true,
+  ga4: false,
+  ga4ContainerId: '',
```

Comment:
Someone one day will want the session IDs for multiple trackers, should this be an array of stream IDs instead?

Reply:
Hmm to allow this case, do you believe it would be fine to turn the `ga4ContainerId` value in the schema into an array of strings ?
 _(afaik gtag.js will use the same `_ga` cookie independently of the number of streams)_

Also I am thinking of adding an option to allow for collecting ga cookies using a different prefix https://developers.google.com/tag-platform/devguides/cookies#rename_cookies .

Reply:
This is most likely going to be used as comparison to GA4 so you'll need to know which measurement ID (my bad, stream ID is a different concept) the session ID is supposed to match with.

So in the options an array is probably fine, but then sadly in the schema I think you'll need array of objects to capture them properly as key value pairs...

Might be able to include the measurement ID as a field and attach a separate entity for each one found, as an alternative? But you'll then have duplicate `_ga` info... :disappointed: